src/server/routes.ts

```diff
@@ -8,104 +8,162 @@ import { Feil } from './utils/Feil';
 import hentAvansertDokumentHtml from './hentAvansertDokumentHtml';
 import { DokumentType } from '../typer/dokumentType';
 import validerDokumentApiData from './utils/valideringer/validerDokumentApiData';
+import { logError, logInfo, logSecure } from '@navikt/familie-logging';
 
 const router = express.Router();
 
 router.get('/status', (_, res) => {
   res.status(200).end();
 });
 
-router.post('/:datasett/dokument/:maalform/:dokumentApiNavn/html', async (req, res) => {
-  const datasett = req.params.datasett as Datasett;
-  const maalform = req.params.maalform as Maalform;
-  const dokumentApiNavn = req.params.dokumentApiNavn;
+router.post(
+  '/:datasett/dokument/:maalform/:dokumentApiNavn/html',
+  async (req: Request, res: Response) => {
+    const datasett = req.params.datasett as Datasett;
+    const maalform = req.params.maalform as Maalform;
+    const dokumentApiNavn = req.params.dokumentApiNavn;
 
-  const dokument: IDokumentData = req.body as IDokumentData;
+    const dokument: IDokumentData = req.body as IDokumentData;
 
-  try {
-    await validerDokumentApiData(datasett, maalform, dokumentApiNavn, DokumentType.DOKUMENT);
-    const html = await hentDokumentHtml(dokument, maalform, dokumentApiNavn, datasett);
-    res.send(html);
-  } catch (error) {
-    if (error instanceof Feil) {
-      return res.status(error.code).send(error.message);
+    logGenereringsrequestTilSecurelogger<IDokumentData>(datasett, dokumentApiNavn, dokument, req);
+    try {
+      await validerDokumentApiData(datasett, maalform, dokumentApiNavn, DokumentType.DOKUMENT);
+      const html = await hentDokumentHtml(dokument, maalform, dokumentApiNavn, datasett);
+      res.send(html);
+    } catch (feil) {
+      if (feil instanceof Feil) {
+        return res.status(feil.code).send(feil.message);
+      }
+
+      logError(`Generering av dokument (html) feilet: ${feil.message}`);
+      logSecure(`Generering av dokument (html) feilet: ${feil}`);
```

Comment:
Skal det ikke logges for "Feil"?

Reply:
Hva er forskjellen på logerror og logsecure?

Reply:
Det er egen logging i Feil erroren. secure kommer til egen plass på kibana med høyere sikkerhetsnivå. Der kan vi logge sensitive data.